Taskfile.yml

```diff
@@ -289,14 +300,14 @@ tasks:
     vars: *x-deploy-vars
     preconditions: *x-deploy-pre
     cmds:
-      - task: cluster-exec
-        vars:
-          HP_TAG: "{{.HP_TAG}}"
-          SSH_TAG: "{{.SSH_TAG}}"
-          ROOT_TOKEN: "{{.ROOT_TOKEN}}"
-          ANSIBLE_VAULT_PATH: "{{.ANSIBLE_VAULT_PATH}}"
-          WORKSPACE: "{{.WORKSPACE}}"
-          HOOK: "{{.HOOK}} pre"
+      # - task: cluster-exec
```

Comment:
Uncomment these - why are they commented out again?  I remember talking about it but can't remember the reason.